pom.xml

```diff
@@ -412,7 +417,7 @@
                         end to end test could run. This can be removed later when dynamic conformance
                         becomes more efficient
                     -->
-                    <argLine>-Xmx4000m -Dfile.encoding=UTF-8</argLine>
+                    <argLine>-Xmx4000m -Dfile.encoding=UTF-8 ${argLine}</argLine>
```

Comment:
What does the `${argLine}` variable doing?

Reply:
Without this change all argLine values provided by JaCoCo are removed.

Reply:
In commit ce7c3fa686ae91c964dc13a408e608ae1c5f17ee work with this variable has been improved.